### PR TITLE
Add easy start with js frameworks post

### PR DESCRIPTION
**Pull request by denar90**
Related to #34 
@scott-w do you want to add something?
Also I would like to have https://github.com/marionettejs/marionette-integrations/pull/36 PR be merged before.

### CODE REVIEW

**Comment by paulfalgout** [comment]
I'm just now seeing this for some reason..  I'm going to look it over shortly.. this is a needed post for sure.

File: source/_posts/easy-start-with-javascript-frameworks.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+  <title>First application with Marionette.js</title>
+</head>
+<body>
+  <div id="app"></div>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.2/jquery.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.3.3/backbone-min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.radio/2.0.0/backbone.radio.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.marionette/3.0.0/backbone.marionette.js"></script>
```
Comment (rafde) [inline source/_posts/easy-start-with-javascript-frameworks.md]:
```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.marionette/3.1.0/backbone.marionette.js"></script>
```

Comment (denar90) [inline source/_posts/easy-start-with-javascript-frameworks.md]: 😄  thanks 👍

**Comment by denar90** [comment]
Should I public it on `medium`?

File: source/_posts/easy-start-with-javascript-frameworks.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+So we will extend `Marionette.Application` class to create our own. We use for this inheritance principle.
+
+```javascript
+var App = Marionette.Application.extend({
+  region: '#app'
+});
+```
+
+Now we want to show something in our region. We have to create view.
```
Comment (scott-w) [inline source/_posts/easy-start-with-javascript-frameworks.md]: "Now, to show something in our region, we need to create a view"

File: source/_posts/easy-start-with-javascript-frameworks.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+ > Note: [Here](http://marionettejs.com/docs/master/marionette.view.html) docs about views.
+
+```javascript
+var View = Mn.View.extend({
+  template: '#template-layout'
+});
+```
+
+Template for View will be placed in scope of body tag. In our small example is used [Underscore template engine](http://underscorejs.org/#template)
```
Comment (scott-w) [inline source/_posts/easy-start-with-javascript-frameworks.md]: "The `template` for `View` is a jQuery selector to a `script` tag in our HTML `body`. For this example, we're using [Underscore's template engine](link).

File: source/_posts/easy-start-with-javascript-frameworks.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+  var app = new App();
+
+  app.start();
+
+})();
+
+```
+
+> Note: All code is put in  [closure](https://developer.mozilla.org/en/docs/Web/JavaScript/Closures) to not let variables get in global scope.
```
Comment (scott-w) [inline source/_posts/easy-start-with-javascript-frameworks.md]: "in a [closure](link) to prevent variables leaking into global scope."

**Comment by scott-w** [comment]
I think publishing on Medium would be a great idea, it's becoming a good common place for general blogging.

File: source/_posts/easy-start-with-javascript-frameworks.md [hunk 5]
```diff
@@ -42,9 +42,9 @@ First of all you have to create `html` page with all needed dependancies.
 > Note: Scripts are loaded using [cdn](https://en.wikipedia.org/wiki/Content_delivery_network), instead of copying files locally.
 
 
-`#app` is a main region where all content using Marionette.js will be rendered.
+`#app` is the main region where all content using Marionette.js will be rendered.
 
-Now we need create some javascript file but before doing that we should look at the [documentation](http://marionettejs.com/docs/v3.1.0/marionette.application.html).
+We need create some javascript file but, before doing that, we should look at the [documentation](http://marionettejs.com/docs/v3.1.0/marionette.application.html).
```
Comment (paulfalgout) [inline source/_posts/easy-start-with-javascript-frameworks.md]: We need to create

**Comment by denar90** [comment]
Thanks guys 👍  I'll prepare medium post and think we can publish posts simultaneously.

**Pull request merged**